src/app/store/effects/VehicleTechRecordModel.effects.ts

```diff
@@ -80,71 +81,51 @@ export class VehicleTechRecordModelEffects {
+        this._technicalRecordService
+          .getTechnicalRecordsAllStatuses(payload.vin, 'all')
+          .pipe(catchError((error) => of(undefined)))
+      );
+
       if (payload.vType === 'PSV' || payload.vType === 'HGV') {
-        requests.push(
-          this._technicalRecordService
-            .getTechnicalRecordsAllStatuses(payload.vin, 'all')
-            .pipe(catchError((error) => of(undefined)))
-        );
         requests.push(
           this._technicalRecordService
             .getTechnicalRecordsAllStatuses(payload.vrm, 'all')
             .pipe(catchError((error) => of(undefined)))
         );
-
-        forkJoin(requests).subscribe((result) => {
-          if (result[0] !== undefined) {
-            requestErrors.push(
-              'A technical record with this VIN already exists, check the VIN or change the existing technical record'
-            );
-          }
-          if (result[1] !== undefined) {
-            requestErrors.push(
-              'A technical record with this VRM already exists, check the VRM or change the existing technical record'
-            );
-          }
-          if (result[0] === undefined && result[1] === undefined) {
-            this.router.navigate([`/technical-record`]);
-          }
-          requests.length = 0;
-          this._store.dispatch(
-            new SetVehicleTechRecordModelVinOnCreateSucess({
-              vin: payload.vin,
-              vrm: payload.vrm,
-              vType: payload.vType,
-              error: requestErrors
-            })
-          );
-        });
-      } else if (payload.vType === 'Trailer') {
-        this._technicalRecordService.getTechnicalRecordsAllStatuses(payload.vin, 'all').subscribe(
-          (result) => {
-            requestErrors.push(
-              'A technical record with this VIN already exists, check the VIN or change the existing technical record'
-            );
-            this._store.dispatch(
-              new SetVehicleTechRecordModelVinOnCreateSucess({
-                vin: payload.vin,
-                vrm: payload.vrm,
-                vType: payload.vType,
-                error: requestErrors
-              })
-            );
-          },
-          (error) => {
-            this.router.navigate([`/technical-record`]);
-          }
-        );
       }
 
+      forkJoin(requests).subscribe(([res1, res2]) => {
+        if (res1 !== undefined) {
+          requestErrors.push(CREATE_PAGE_LABELS.CREATE_VIN_LABEL_ERROR);
+        }
+        if (res2 !== undefined) {
+          requestErrors.push(CREATE_PAGE_LABELS.CREATE_VRM_LABEL_ERROR);
+        }
+        if (res1 === undefined && res2 === undefined) {
+          const vehicleTechRecord = {} as VehicleTechRecordModel;
+          const vrms = { vrm: payload.vrm, isPrimary: true } as VrmModel;
+          vehicleTechRecord.vin = payload.vin;
+          vehicleTechRecord.vrms = [vrms];
+          vehicleTechRecord.techRecord = [{} as TechRecord];
```

Comment:
`vehicleTechRecord.techRecord = [{
  statusCode: 'provisional'
} as TechRecord];`

Good job by the way